cluster/v1/0000_00_clusters.open-cluster-management.io_managedclusters.crd.yaml

```diff
@@ -35,14 +35,15 @@ spec:
     schema:
       openAPIV3Schema:
         description: "ManagedCluster represents the desired state and current status
```

Comment:
Suggestion: "ManagedCluster represents the desired state and current status
          of a managed cluster. ManagedCluster is a cluster-scoped resource. The name
          is the cluster UID. \n The cluster join process is a double opt-in
          process. See the following join process steps: \n 1. The agent on the managed cluster creates a CSR on the hub
          with the cluster UID and agent name. 2. The agent on the managed cluster
          creates a ManagedCluster on the hub. 3. The cluster admin on the hub cluster approves
          the CSR for the UID and agent name of the ManagedCluster. 4. The cluster
          admin sets the spec.acceptClient of the ManagedCluster to true. 5. The cluster
          admin on the managed cluster creates a credential of the kubeconfig for
          the hub cluster. \n After the hub cluster creates the cluster namespace, the klusterlet agent
          on the ManagedCluster pushes the credential to the hub cluster to use against the
          kube-apiserver of the ManagedCluster."